language-support/java/bindings/src/main/java/com/daml/ledger/javaapi/data/Template.java

```diff
@@ -5,12 +5,10 @@
 
 import com.daml.ledger.javaapi.data.codegen.CreateAnd;
 
-public abstract class Template {
+public abstract class Template extends com.daml.ledger.javaapi.data.codegen.DamlRecord<Template> {
```

Comment:
We would rather add a (currently phantom but probably not soon based on decoder developments) type parameter to `Template`, but that is a bigger change, and this is fine as a workaround for now.